### PR TITLE
Update tenant-rename.md

### PR DESCRIPTION
**Pull request by WayneEwington**
Removed Power Apps limitations which has now been addressed and addressing key feedback items from Private Preview customers:
* Removed warning about having a hyphen in the MOERA Domain as the UX blocks this.
* Clarified and provided additional information and warning about using the provided "special" hyperlink to add an additional MOERA Domain and the type of domain required.
* Provided additional instructions on how to remove previous versions of TAPS.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/tenant-rename.md [hunk 1]
```diff
@@ -112,28 +110,33 @@ It's now possible to change the SharePoint domain name for your organization in
 
 2. Go to the [Custom domain names page in the Azure AD admin center](https://aka.ms/SPORenameAddDomain).
 
+    > [!IMPORTANT]
+    > Use the hyperlink [https://aka.ms/SPORenameAddDomain](https://aka.ms/SPORenameAddDomain) provided above. Failiure to use this hyperlink will prevent you from successfully adding a custom domain.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/tenant-rename.md:114]:
```suggestion
    > You must use the hyperlink [https://aka.ms/SPORenameAddDomain](https://aka.ms/SPORenameAddDomain). Failure to use this hyperlink will prevent you from successfully adding a custom domain.
```

File: SharePoint/SharePointOnline/tenant-rename.md [hunk 2]
```diff
@@ -112,28 +110,33 @@ It's now possible to change the SharePoint domain name for your organization in
...
+
 3. Select **Add custom domain**.
 
 4. In the **Custom domain name** box, add the full new “.onmicrosoft.com” domain, and then select **Add domain**.
 
     ![Custom domain name pane](media/custom-domain-name.png)
 
     > [!IMPORTANT]
-    > Do NOT include any hyphens (-) in the new domain. They aren't supported in SharePoint.
+    > When adding the domain this needs to be a “onmicrosoft.com” domain. For example, if you want to rename your tenant to fabrikam.sharepoint.com, the domain that you enter should be fabrikam.onmicrosoft.com. Adding a “onmicrosoft.com” domain does not need to be purchased or require any public DNS registration.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/tenant-rename.md:123]:
```suggestion
    > When adding the domain, it needs to be an “onmicrosoft.com” domain. For example, if you want to rename your tenant to fabrikam.sharepoint.com, the domain that you enter should be fabrikam.onmicrosoft.com. The “onmicrosoft.com” domain does not need to be purchased to be added and does not require any public DNS registration.
```

File: SharePoint/SharePointOnline/tenant-rename.md [hunk 3]
```diff
@@ -112,28 +110,33 @@ It's now possible to change the SharePoint domain name for your organization in
...
  
 5. Make sure you get a confirmation message. If the domain isn't available, try a different domain. 
  
 6. After getting a confirmation that the domain was added successfully, you might see a message that the properties could not be found. Select the message to refresh domain references.
 
     > [!WARNING]
-    > Do NOT add any other domains. Do NOT configure the new domain as the initial domain.
+    > Do NOT add any other domains.
+    > Do NOT configure the new domain as the initial domain.
+    > If after adding the domain you are prompted to create a new TXT record with your domain name registrar, then the domain has NOT been added correctly and you will NOT be able to perform a rename. If you are prompted then you will need to delete the invalid domain and return to Step 2 above.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/tenant-rename.md:132]:
```suggestion
    > If, after adding the domain, you are prompted to create a new TXT record with your domain name registrar, then the domain has NOT been added correctly and you will NOT be able to perform a rename. If you are prompted, you will need to delete the invalid domain and return to the previous Step 2.
```

File: SharePoint/SharePointOnline/tenant-rename.md [hunk 4]
```diff
@@ -112,28 +110,33 @@ It's now possible to change the SharePoint domain name for your organization in
...
  
 7. Confirm that your domain has been added to the list. 
 
     ![Domain status verified](media/domain-verified.png)
 
     > [!IMPORTANT]
-    > The domain name must have the status of "Verified."
+    > The domain name must have the status of "Verified". If the status is NOT "Verified" then you will NOT be able to perform a rename.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/tenant-rename.md:139]:
```suggestion
    > The domain name must have the status of "Verified". If the status is NOT "Verified", you will NOT be able to perform a rename. 
```

File: SharePoint/SharePointOnline/tenant-rename.md [hunk 5]
```diff
@@ -163,6 +166,9 @@ It's now possible to change the SharePoint domain name for your organization in
 
     `Start-SPOTenantRename -DomainName "fabrikam" -ScheduledDateTime "2021-12-31T10:25:00"`
 
+    > [!NOTE]
+    > If the PowerShell command Start-SPOTenantRename is not found or nothing is returned, then ensure that you have installed the latest SharePoint Online Management Shell. If required, before installing the latest version you may need to uninstall all previous versions by running `Uninstall-Module Microsoft.Online.SharePoint.PowerShell -Force -AllVersions`
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/tenant-rename.md:170]:
```suggestion
    > If the PowerShell command Start-SPOTenantRename is not found or nothing is returned, ensure that you have installed the latest SharePoint Online Management Shell. If required, before installing the latest version, you might need to uninstall all previous versions by running `Uninstall-Module Microsoft.Online.SharePoint.PowerShell -Force -AllVersions`.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @WayneEwington, kindly commit the suggested edits if you agree with them. Many thanks.

**Comment by WayneEwington** [comment]
@JohanFreelancer9 changes have been committed

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**